tools/depends/target/crossguid/CMakeLists.txt

```diff
@@ -1,12 +1,12 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
Better change direct to 3.4 (or 3.5) to have equal to the base.

Reply:
This PR is about switching to C++14, not about unifying `cmake_minimum_required`.
Version 3.1 is enough for this change.